src/EPR.Calculator.API/Exporter/CalcResultsExporter.cs

```diff
@@ -1,12 +1,58 @@
 ﻿using EPR.Calculator.API.Models;
+using EPR.Calculator.API.Services;
+using EPR.Calculator.API.Utils;
+using System;
+using System.Text;
 
 namespace EPR.Calculator.API.Exporter
 {
     public class CalcResultsExporter : ICalcResultsExporter<CalcResult>
     {
+        private readonly IBlobStorageService _blobStorageService;
+        public CalcResultsExporter(IBlobStorageService blobStorageService)
+        {
+            _blobStorageService = blobStorageService;
+        }
         public void Export(CalcResult results)
         {
-            // Code to export the File to Csv
+            var csvContent = new StringBuilder();
```

Comment:
Please create a RunDetailsConstant and add all the columns to it.

Reply:
updated